gammapy/modeling/models/core.py

```diff
@@ -1164,19 +1164,29 @@ def positions(self):
 
         return SkyCoord(positions)
 
-    def to_regions(self):
+    def to_regions(self, x_width=None, **kwargs):
         """Return a list of the regions for the spatial models.
 
+        Parameters
+        ----------
+        x_width : float, optional
+            Number of :math:`sigma` for `GaussianSpatialModel`
+            or `x_r_0` for `GeneralizedGaussianSpatialModel`
+            If not specified, the defaults for the models will be used.
+        kwargs : dict, optional
+            Keyword arguments passed to `Spatial_model.to_region`
```

Comment:
Can we make sure its the correct link?

Otherwise my suggestion would be:

```suggestion
            Keyword arguments passed to ``model.spatial_model.to_region``.
```